apps/src/code-studio/pd/workshop_dashboard/EnrollmentsPanel.jsx

```diff
@@ -4,10 +4,16 @@ import PropTypes from 'prop-types';
 import {Button} from 'react-bootstrap';
 import moment from 'moment';
 import MoveEnrollmentsDialog from './components/move_enrollments_dialog';
+import EditEnrollmentNameDialog from './components/edit_enrollment_name_dialog';
 import Spinner from '../components/spinner';
 import WorkshopEnrollment from './components/workshop_enrollment';
 import WorkshopPanel from './WorkshopPanel';
 
+const BUTTON_NAMES = {
```

Comment:
Wasn't sure what appropriate casing is here.

Reply:
this seems reasonable to me

Reply:
We don't enforce this element of our styleguide, so this is fine.

If you'd like a general rule to apply, we default to the Google JavaScript styleguide.  Their [section on constants](https://google.github.io/styleguide/javascriptguide.xml#Constants) isn't entirely clear. One  suggests you prefer this case for the primitive/terminal values, more like:

```js
const buttonNames = {
  MOVE_ENROLLMENT: 'moveEnrollment',
  EDIT_ENROLLMENT_NAME: 'editEnrollmentName'
};

// So that you can use it like so:
buttonNames.EDIT_ENROLLMENT_NAME
``` 

On the other hand, these values, while semantically constant, are not actually made constant by the `const` keyword in this case so that might be misleading. (`buttonNames` cannot be reassigned, but the object it points to can still be modified.)  The AirBnB styleguide (which we usually turn to for React style guidance) [mostly agrees with what you've done here](https://github.com/airbnb/javascript#naming--uppercase).  Except that, as this is not an _exported_ constant, they wouldn't have you const-case any part of it.

There's a decent argument to be made in JavaScript that you should just use less structure for constants like this:

```js
const MOVE_ENROLLMENT_BUTTON_NAME = 'moveEnrollment';
const EDIT_ENROLLMENT_BUTTON_NAME = 'editEnrollment';
```

...at least until you feel you're getting significant value from more structure.

In the end, your call. :grin:

Reply:
Your suggestion makes sense to me -- I updated, and exported those constants to the test as well such that if they were to get updated they wouldn't have to get updated in the test as well.